policies/design-process.md

```diff
@@ -1,160 +1,164 @@
-iterate back to the "Refine and document" step to incorporate feedback from the
-OTC. In some cases more significant rework may be required, in which case the
-OTC may choose to step back to any previous step in this design process.
-
-In some cases the OTC may approve a solution which has certain details
-deliberately omitted in order to defer work on those details until a later stage
-of the implementation project. In such cases this process iterates back to the
-"Refine and document" stage to fill in those details when they are eventually
-required, and the subsequent amended design document must be approved again by
-the OTC.
-
-Transparency and Community Involvement
---------------------------------------
-
-It should be possible for community members to see the progress of designs
-through the process and to pass comment on the proposed design prior to its
-eventual agreement by OTC. As a minimum:
-
-* The draft design document containing the candidate solutions should be
-available as a pull request for public comment for at least one week prior to
-the selection of the solution. In addition to that, community members may be
-offered an invite to join the solution selection meeting.
+Design Process Policy
+=====================
+
+Objectives
+----------
+
+The objective of the design process is to increase the quality of the software
+engineering process. The production of design documents confers the following
+benefits:
+
+  - Prior to implementation, the present understanding of the problem domain is
+    documented in a readily accessible fashion. The understanding of other
+    OpenSSL contributors is enhanced, as is their ability to identify any
+    potential issues in the design, or to make related code contributions.
+
+  - After implementation, the design document serves as documentation of
+    the architectural and design decisions and rationale which served as the
+    basis for the implementation of the relevant functionality. This is
+    beneficial both to contributors which wish to understand the relevant code,
+    or evolve the implementation, but also to contributors who wish to implement
+    related functionality or understand non-obvious rationales behind given
+    design decisions.
+
+It is recognised that designs will often necessarily change once implementation
+begins. In the majority of cases, the understanding of the problem domain will
+evolve and improve after implementation begins and this will indicate further
+changes to the design. This is an iterative process of progressive refinement.
+It is an explicit objective of this policy to support and encourage this
+agile-style process, as opposed to a waterfall-style process in which designs
+must be approved and finalised prior to implementation.
+
+Moreover, it may be hard to obtain explicit approval for a design from the OTC
+where there design falls out of the domains of the OTC members' expertise. This
```

Comment:
I would just drop this sentence as there are also other reasons why graded system of reviews is good idea.